Functions.Templates/ProjectTemplates.nuspec

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.AzureFunctions.ProjectTemplates</id>
+    <id>Microsoft.Azure.Functions.ProjectTemplates</id>

```

Comment:
This may have to use the existing `Microsoft.Azure.WebJobs.*` namespace

Reply:
done.